src/content/docs/en/guides/migrate-to-astro/from-nuxtjs.mdx

```diff
@@ -2,68 +2,671 @@
+  return pokemons.map(({ name }) => ({
+      params: { name },
+    }))
+}
+
+// ...
+---
+
+<!-- Your template here -->
+```
+
+### Nuxt Data Fetching to Astro
+
+Nuxt has two methods of fetching server-side data:
+
+- [`asyncData` options API](https://nuxtjs.org/docs/features/data-fetching/#async-data)
+- [`fetch` hook](https://nuxtjs.org/docs/features/data-fetching/#the-fetch-hook)
+
+Astro simplifies this to simply fetch data inside of your page's code fence.
+
+Migrate the following:
+
+```vue title="pages/index.vue"
+{
+  // ...
+  async asyncData() {
+    const res = await fetch("https://pokeapi.co/api/v2/pokemon?limit=151")
+    const resJson = await res.json();
+    const pokemons = resJson.results;
+    return {
+      pokemons,
+    }
+  },
+}
+```
+
+To a code fence without a wrapper function:
+
+```astro title="src/pages/index.astro"
+---
+const res = await fetch("https://pokeapi.co/api/v2/pokemon?limit=151")
+const resJson = await res.json();
+const pokemons = resJson.results;
+---
+
+<!-- Your template here -->
+```
+
+### Nuxt Styling to Astro
+
+Nuxt utilizes Vue's component styling to generate a page's style.
+
+```vue title="pages/index.vue"
+<template>
+	<!-- Your template here -->
+</template>
+
+<script>
+	// Your server logic here
+</script>
+
+<style scoped>
+    .class {
+        color: red;
+    }
+</style>
+```
+
+ Similarly, in Astro you can drop in a `<style>` element in your page's template to style the component:
+
+```astro title="src/pages/index.vue"
+---
+// Your server logic here
+---
+
+<style>
+    .class {
+        color: red;
+    }
+</style>
+```
+
+#### Global Styling
+
+`<style>`  tags are `scoped` by default in Astro. To make a `<style>` tag global, mark it with the `is:global` attribute:
+
+```astro title="src/pages/index.vue"
+<style is:global> 
+	p {
+		color: red;
+	}
+</style>
+```
+
+
+
+#### Pre-processor support
+
+[Astro supports the most popular CSS preprocessors right out of the box](/en/guides/styling/#css-preprocessors) by installing them as a dev dependency. For example, to use SCSS you'd run:
+
+```shell
+npm install -D sass
+```
+
+After doing so, you're then able to use `.scss` or `.sass` styled without modification from your Vue components.
+
+```astro title="src/layouts/Layout.astro"
+<p>Hello, world</p>
+<style lang="scss">
+p {
+   color: black;
+   
+   &:hover {
+       color: red;
+   }
+}
+</style>
+```
+
+See more about [Styling in Astro](/en/guides/styling/).
+
+### Nuxt Image Plugin to Astro
+
+Convert any [Nuxt `<nuxt-img/>` or `<nuxt-picture/>` components](https://image.nuxtjs.org/components/nuxt-img) to [Astro's own image integration components](/en/guides/images/#astros-image-integration), or with a standard HTML `<img>`. See a [full list of component attributes](/en/guides/integrations-guide/image/#usage) available for Astro's `<Image />` and `<Picture />` components, and note that several will differ from Nuxt's attributes. 
+
+```astro title="src/pages/index.astro"
```

Comment:
I'm not sure it's worth it - it would just be a singular <nuxt-img> tag like:
```
 <nuxt-img src="/images/brand/logo.png" alt="The Astro Logo" />
```
They already know what `nuxt-img` looks like, and I don't want to construct an entire Vue component just to show this tag.